src/components/DiagramFrame/DiagramCanvas/Diagram.tsx

```diff
@@ -56,7 +56,7 @@ export const Diagram: React.FC<DiagramProps> = ({
     return viewVisible[viewName]
```

Comment:
Most of these changed files are only linting. The only substantive changes are in `VisualizationEditor.tsx`